### ISSUE TITLE
ComedyCentral downloader broken

### ISSUE THREAD

**Issue by donald-jackson** [rg3/youtube-dl#8972]
Running latest version as of 2016-03-25 - no longer able to download videos from Comedy Central.

Mac OS X 10.11

youtube-dl -f 1700 http://www.cc.com/full-episodes/0d8i55/the-daily-show-with-trevor-noah-march-22--2016----taavi-roivas-season-21-ep-21082
[ComedyCentral] the-daily-show-with-trevor-noah-march-22--2016----taavi-roivas-season-21-ep-21082: Downloading webpage
[ComedyCentral] f6e4b53d-2fcc-4e53-95f8-72c4472f8d8b: Downloading info
[ComedyCentral] 8931fed9-77fd-4bf7-ae61-ba17280d9cdb: Extracting information
[ComedyCentral] 8931fed9-77fd-4bf7-ae61-ba17280d9cdb: Downloading video urls
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/bin/youtube-dl/__main__.py", line 19, in <module>
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 412, in main
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 402, in _real_main
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1719, in download
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 668, in extract_info
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 320, in extract
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/mtv.py", line 219, in _real_extract
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/mtv.py", line 183, in _get_videos_info
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/mtv.py", line 190, in _get_videos_info_from_url
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/mtv.py", line 135, in _get_video_info
AttributeError: 'NoneType' object has no attribute 'strip'

**Comment by dstftw** [comment]
**Post the full output of youtube-dl when run with `-v`**, i.e. **add** `-v` flag to **your command line**, copy the **whole** output and post it in the issue body wrapped in \`\`\` for better formatting. It should look similar to this:
```
$ youtube-dl -v <your command line>
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'http://www.youtube.com/watch?v=BaW_jenozKcj']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2015.12.06
[debug] Git HEAD: 135392e
[debug] Python version 2.6.6 - Windows-2003Server-5.2.3790-SP2
[debug] exe versions: ffmpeg N-75573-g1d0487f, ffprobe N-75573-g1d0487f, rtmpdump 2.4
[debug] Proxy map: {}
...
```
**Do not post screenshots of verbose log only plain text is acceptable.**

The output (including the first lines) contains important debugging information. Issues without the full output are often not reproducible and therefore do not get solved in short order, if ever.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2016.03.25. See [our FAQ](https://yt-dl.org/update) if you need help updating.
